src-electron/db/query-config.js

```diff
@@ -211,6 +211,14 @@ async function insertOrUpdateAttributeState(
       attributeId,
       clusterRef
     )
+  if (
+    staticAttribute.storagePolicy ==
+    dbEnum.storagePolicy.attributeAccessInterface
+  ) {
+    staticAttribute.storagePolicy = dbEnum.storageOption.external
```

Comment:
This is mixing up _storage policy_ (which can be "any" or "attributeAccessInterface") and _storage option_, which can be "external", "ram", "nvm".  See also https://github.com/project-chip/zap/issues/367

So what I would suggest is that instead of storing "storage option" values in a field that stores storage policy, instead we just use a local variable for the storage option to use for our db insert.

Incidentally, when I was looking over this setup, the `resolve` method on `storagePolicy` in db-enum.js is broken: it has `storagePolicy.attributeAccess` where it should have `storagePolicy.attributeAccessInterface`, no?  Not directly relevant to this PR, but we should fix it here or in a followup.

Reply:
agree, I will make a follow up PR. Thank you.